platform/logging/layouts/JsonLayout.ts

```diff
@@ -0,0 +1,30 @@
+import { Schema, typeOfSchema } from '../../types';
+import { LogRecord } from '../LogRecord';
+import { Layout } from './Layouts';
+
+const createSchema = ({ literal, object }: Schema) => {
+  return object({
+    kind: literal('json')
+  });
+};
+
+const schemaType = typeOfSchema(createSchema);
+/** @internal */
+export type JsonLayoutConfigType = typeof schemaType;
+
+/**
+ * Layout that just converts `LogRecord` into JSON string.
+ * @internal
+ */
+export class JsonLayout implements Layout {
+  static createConfigSchema = createSchema;
+
+  format(record: LogRecord): string {
+    // Per spec `timestamp.toJSON()` uses `timestamp.toISOString()` under the hood.
+    // See http://www.ecma-international.org/ecma-262/5.1/#sec-15.9.5.44
+    return JSON.stringify({
```

Comment:
Output looks like this:
```bash
{"timestamp":"2017-07-31T10:39:40.411Z","level":{"id":"info","value":5},"context":"elasticsearch.client.admin","message":"cluster client stopped"}
```
But I don't know use case for JSON output, really (cc @epixa, @tylersmalley).  Maybe we need something like this instead:

```typescript
format({ timestamp, level, context, message, error, meta }: LogRecord): string {
    // Per spec `timestamp.toJSON()` uses `timestamp.toISOString()` under the hood.
    // See http://www.ecma-international.org/ecma-262/5.1/#sec-15.9.5.44
    return JSON.stringify({
      '@timestamp': timestamp,
      level: level.id,
      context,
      message,
      error: error && error.message,
      meta
    });
  }
```
```bash
{"@timestamp":"2017-07-31T10:35:49.230Z","level":"info","context":"elasticsearch.client.admin","message":"cluster client stopped"}
```